exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -45,11 +45,13 @@ drill.client: {
   supports-complex-types: true
 }
 
-// Directory is used as base for temporary storage of Dynamic UDF jars.
-// Set this property if you want to have custom temporary directory, instead of generated at runtime.
-// By default ${DRILL_TMP_DIR} is used if set.
-// drill.tmp-dir: "/tmp"
-// drill.tmp-dir: ${?DRILL_TMP_DIR}
+// Location Drill uses for temporary files, such as downloaded dynamic UDFs jars.
```

Comment:
Can we leave it that, if blank, it will use the system-generated directory as you have in 1.9? This means we'll still need your getTmpDir method. To do the check for blank.

Also, as it turns out, we already have a temp dir setting used elsewhere. Look in drill-override-example.conf under drill.sys.store.provider.local.path. It uses "/tmp/drill". Then it adds subdirs for storage plugins and what-not.

Can we use "/tmp/drill/udf" as the base temp storage location for udfs to be consistent? Then we'd add the cluster-id, etc. as in your "cluster-temp-dir" below.

And, yes, we have lots of places where we specify the temp file system (UDFs, the one cited above, the (unused) cache.hazel, the temp dirs for spilling, ...). We do need to create a project (later) to rationalize all of this...